lib/build.js

```diff
@@ -258,6 +263,13 @@ export default {
         env[key] = replace(env[key], target.env);
       });
 
+      if (variant) {
+        const copy = {};
+        Object.assign(copy, target);
+        Object.assign(copy, target.variants[variant]);
+        target = copy;
```

Comment:
Please not that `Object.assign` accepts spread arguments, meaning you can have as many arguments as you want and any keys later will override the earlier ones.